tools/integration_tests/util/setup/setup.go

```diff
@@ -394,3 +394,55 @@ func AreBothMountedDirectoryAndTestBucketFlagsSet() bool {
 	log.Print("Not running mounted directory tests as both --mountedDirectory and --testBucket flags are not set.")
 	return false
 }
+
+func separateBucketAndObjectName(bucket, object string) (string, string) {
+	bucketAndObjectPath := strings.SplitN(bucket, "/", 2)
+	bucket = bucketAndObjectPath[0]
+	object = path.Join(bucketAndObjectPath[1], object)
+	return bucket, object
+}
+
+func SetBucketAndObjectBasedOnTypeOfMount(object string) (string, string) {
```

Comment:
Nit: Rename to GetBucketAndObjectBasedOnTypeOfMount as this function now returns object and bucket